users_permissions/README.md

```diff
@@ -50,13 +50,19 @@ For the purpose of this demo you have to create your own Atlas App Service. Foll
 1. Go into folder '\assets\atlas_app':
 
     `cd assets/atlas_app`
-
 1. Deploy the app to Atlas App Services:
 * IMPORTANT: Before to push the app make sure the cluster name is the same like the cluster in your account. Go to "\assets\atlas_app\data_sources\mongodb-atlas\config.json" and set the json field `clusterName`.
 Then run this command:
 
     `realm-cli push --yes`
-
+1. Then go to the UI and manually allow `Custom User Data` in **Authentication**. Select the following fields:
```

Comment:
This section will be removed after this issue is fixed https://jira.mongodb.org/browse/HELP-42773